### PR TITLE
[FAD-84] - Add artist pool card component and minor fixes

### PR DESCRIPTION
**Pull request by luansievers**
### Description

> Add artist pool card component and add fixes to other components/pages (Comments added for those fixes)
> Fix chip component
> Add progress component

image

### Checklist

- [ ] `CHANGELOG` has been updated (_if appropriate_)
- [ ] Environment variables updated (_if appropriate_)

### CODE REVIEW

File: packages/client3/components/dashboard/my-open-pool/artist-pool.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ type Props = {
 
 export function ArtistPool({ isVerified, onButtonClick }: Props) {
   return (
-    <div className="grid place-items-center bg-green-100 pt-14 pb-12">
+    <div className="grid place-items-center rounded-lg bg-green-100 pt-14 pb-12">
```
Comment (luansievers) [inline packages/client3/components/dashboard/my-open-pool/artist-pool.tsx:10]: Missing class

File: packages/client3/components/dashboard/pool-card/pool-card.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          {formatCrypto(totalSuppliedAmount)}
+        </BodyText>
+      </div>
+      {/* pt-[28px] or pt-[20px] */}
+      {/* <div className="flex-1 pt-[20px]">
+        <BodyText size="normal" className="text-light-40">
+          July 30, 2022
+        </BodyText>
+        <Caption className="pt-[3px] text-dark-50">13 days left</Caption>
+      </div> */}
```
Comment (luansievers) [inline packages/client3/components/dashboard/pool-card/pool-card.tsx:48]: when we have a closing date

File: packages/client3/components/dashboard/total/dashboard-total.stories.tsx [hunk 3]
```diff
@@ -3,20 +3,20 @@ import { BigNumber } from "ethers";
...
 import { SupportedCrypto } from "@/lib/graphql/generated";
 
-import { DashBoardTotal } from "./index";
+import { DashboardTotal } from "./index";
 
 export default {
   title: "FAD/Components/Dashboard/Total",
-  component: DashBoardTotal,
-} as ComponentMeta<typeof DashBoardTotal>;
+  component: DashboardTotal,
```
Comment (luansievers) [inline packages/client3/components/dashboard/total/dashboard-total.stories.tsx:10]: change from `DashBoard` to `Dashboard`

File: packages/client3/components/dashboard/total/dashboard-total.tsx [hunk 4]
```diff
@@ -21,7 +21,7 @@ export function DashBoardTotal({
     <div
       className={clsx(
         "grid grid-cols-4 gap-4 bg-green-100 px-7 py-12",
-        "bg-img-free-artist bg-center bg-no-repeat"
+        "rounded-lg bg-img-free-artist bg-center bg-no-repeat"
```
Comment (luansievers) [inline packages/client3/components/dashboard/total/dashboard-total.tsx:24]: missing class

File: packages/client3/components/design-system/chip/chip.tsx [hunk 5]
```diff
@@ -13,13 +15,15 @@ export function Chip({ children, className, type = "completed" }: ChipProps) {
...
+          "bg-green-90": type == "completed",
+          "bg-accent-3": type == "failed",
         },
         className
       )}
     >
-      <div className="text-sm text-light-4">{children}</div>
+      <BodyText className="text-light-40" size="small">
+        {children}
+      </BodyText>
```
Comment (luansievers) [inline packages/client3/components/design-system/chip/chip.tsx:26]: Fix chip component

File: packages/client3/components/design-system/progress/progress.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import clsx from "clsx";
+
+interface ProgressProps {
+  percentage: number;
+  type?: "completed" | "failed" | undefined;
+}
+
+export function Progress({ percentage, type = "completed" }: ProgressProps) {
```
Comment (luansievers) [inline packages/client3/components/design-system/progress/progress.tsx:8]: Add progress component

File: packages/client3/components/design-system/snackbar/snackbar.stories.tsx [hunk 7]
```diff
@@ -12,7 +12,7 @@ export default {
 export const SnackbarStory: ComponentStory<typeof Snackbar> = (args) => {
   const notify = () => {
     toast(args.message, {
-      type: args.type,
+      type: args.type as TypeOptions,
```
Comment (luansievers) [inline packages/client3/components/design-system/snackbar/snackbar.stories.tsx:15]: Was getting typescript error when committing, need to cast the attribute

**Review by luansievers — COMMENTED** [review]
Added comments for some changes

**Review by v1n33th — APPROVED** [review]

**Review by sachasmart-weavik — APPROVED** [review]

**Pull request merged**